lib/ingestor-api/index.ts

```diff
@@ -62,6 +62,7 @@ export class StacIngestor extends Construct {
       stage: props.stage,
       endpointConfiguration: props.apiEndpointConfiguration,
       policy: props.apiPolicy,
+      ingestorDomainNameOptions: props.ingestorDomainNameOptions,
```

Comment:
We had missed this which meant we weren't _actually_ using the domain name option for the ingestor. I added it. I tested in https://github.com/MAAP-Project/maap-eoapi/pull/15 that it works.